### DISCUSSION TITLE
Button to set default "Preview Markdown" mode

### DISCUSSION THREAD

**Discussion by hahahumble** [massCodeIO/massCode#249] [Ideas]
Button to set default "Preview Markdown" mode.
I‘ve set to "Preview Markdown" mode, but when I restart massCode, it automatically goes back to "Hide Preview Markdown" mode, I hope there is a button let us set the default "Preview Markdown" mode. Thanks.

Version: 3.3.0

**Comment by Faq** [comment]
duplicate of #46 
@hahahumble go to linked discussion and upvote it.
